src/components/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Todo } from '../types/Types';
+import { TodoItem } from './TodoItem';
+
+type TodoListProps = {
+  todos: Todo[];
+};
+
+export const TodoList = ({ todos }: TodoListProps) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => {
```

Comment:
If you feel like going fancy :) 
`{todos.map(todo => ( <TodoItem key={todo.id} todo={todo} /> ))}`